datafusion/sqllogictest/test_files/select.slt

```diff
@@ -848,6 +848,11 @@ statement error DataFusion error: Error during planning: EXCLUDE or EXCEPT conta
 SELECT * EXCLUDE(a, a)
 FROM table1
 
+# if EXCEPT all the columns, query should still succeed but return empty
```

Comment:
thanks @Jefffrey i would also add more tricky test cases, where column names are in different order, duplicated.
like
```
SELECT * EXCEPT(d, b, c, a, a, b, c, d)
FROM table1
```

Reply:
Sure thing, added tests to cover this